tests/tt_metal/tt_metal/test_kernels/compute/reduce_h.cpp

```diff
@@ -4,22 +4,35 @@
 
 #include <cstdint>
 
-#include "debug/dprint.h"
-
 #include "compute_kernel_api/reduce.h"
-#include "compute_kernel_api/eltwise_binary.h"
+
+template<bool not_at_start, PoolType reduce_type = REDUCE_OP, ReduceDim reduce_dim = REDUCE_DIM>
+ALWI void dummy_init(uint32_t icb = 0, uint32_t icb_scaler = 1, uint32_t ocb = 16)
+{
+#ifdef SHORT_INIT
+    UNPACK(( llk_unpack_AB_hw_configure_disaggregated<DST_ACCUM_MODE>(icb, icb_scaler) ));
+
+    MATH(( llk_math_pack_sync_init<DST_ACCUM_MODE>() ));
+    MATH(( llk_math_hw_configure_disaggregated() ));
+
+    PACK(( llk_pack_init() ));
+    PACK(( llk_pack_dest_init<false, DST_ACCUM_MODE>() ));
+#endif
+    PACK(( llk_pack_reduce_config_v2<reduce_dim, not_at_start, false, DST_ACCUM_MODE>(ocb) ));
```

Comment:
What is the purpose of this dummy init and secondly the bool not_at_start seems contradictory, as this function llk_pack_reduce_v2 takes an argument named at_start.

Reply:
The purpose is to test the 'at_start' parameter. The LLK you mentioned is passed not_at_start just as a negative value from at_start. You can see that in dummy_init call the parameter passed is !at_start. Does it make sense?

Reply:
May be add a comment explaining it, I mean the function takes a boolean which says if it is the start or not, now you are passing it the opposite of it, what if you just revert it with ! at the moment of passing ?

Reply:
Good point, I've added the comments in the kernels.